src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7418,4 +7430,43 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_ParamsArrayInLambdaOnly_Title" xml:space="preserve">
     <value>Parameter has params modifier in lambda but not in target delegate type.</value>
   </data>
+  <data name="ERR_BadExtensionUnderlyingType" xml:space="preserve">
+    <value>Extension '{0}' cannot augment underlying type '{1}'. The extension underlying type may not be dynamic, a pointer, a nullable reference type, a ref struct or an extension.</value>
+  </data>
+  <data name="ERR_StaticBaseTypeOnInstanceExtension" xml:space="preserve">
+    <value>Instance extension '{0}' cannot augment underlying type '{1}' because it is static.</value>
+  </data>
+  <data name="ERR_OnlyBaseExtensionAllowed" xml:space="preserve">
+    <value>'{0}' is not an un-annotated extension type.</value>
```

Comment:
>'{0}' is not an un-annotated extension type. [](http://example.com/codeflow?start=11&length=44)

I am not sure what is an "annotated extension type". Could we simply say "is not an extension type"?